netconf/transport_junos.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2018, Juniper Networks, Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Transport JunOS provides the ability to communicate with JunOS via local shell
+NETCONF interface (xml-mode netconf need-trailer).
+*/
+package netconf
+
+import (
+	"os/exec"
+)
+
+// TransportJunOS maintains the information necessary to communicate with JunOS
+// via local shell.
+type TransportJunOS struct {
```

Comment:
Can you please name this Junos?  Juniper never capitalized the O or S in Junos.

Reply:
fixed with 73f56cb0c96371b905d4a8f82ff98884c13beecf